### PR TITLE
Adding more target="_blank" to links

### PR DESCRIPTION
**Pull request by jkeck**
* IIIF Links
* WAS Viewer Memento Links

@snydman reported this when testing the new IIIF drag and drop links.

### CODE REVIEW

File: spec/features/was_seed_viewer_spec.rb [hunk 1]
```diff
@@ -31,6 +31,10 @@
       page.first('.sul-embed-was-thumb-item').click
       expect(page.find('.active').text).to eq('29-Nov-2012')
     end
+
+    it 'links to the momento URI with a blank target' do
```
Comment (ndushay) [inline spec/features/was_seed_viewer_spec.rb]: memento, not momento

**Comment by ndushay** [comment]
LGTM after you fix the typo.

File: spec/features/was_seed_viewer_spec.rb (hunk 1 above)
Comment (jkeck) [inline spec/features/was_seed_viewer_spec.rb]: Thanks! I thought it was a play on "moment".  😆

**Comment by ndushay** [comment]
👍

**Pull request merged**